docs/framework/Release-Notes/01-January-Security-and-Quality-Rollup

```diff
@@ -0,0 +1,298 @@
+<h2>January Security and Quality Rollup</h2>
+<em> released January 9, 2024</em>
+<h3>Summary of What's New in this Release</h3>
+<h4>Security improvements</h4>
+<h5>CVE-2023-36042 – .NET Framework Denial of Service Vulnerability</h5>
+This security update addresses a security feature bypass vulnerability detailed in <a href="https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE-2023-36042" rel="nofollow">CVE 2023-36042</a>.
+<h5>CVE-2024-0056 – .NET Framework Security Feature Bypass Vulnerability</h5>
+Microsoft is releasing this security advisory to provide information about a vulnerability in .NET's System.Data.SqlClient and Microsoft.Data.SqlClient NuGet Packages. This advisory also provides guidance on what developers can do to update their applications to address this vulnerability.
+
+A vulnerability exists in the Microsoft.Data.SqlClient and System.Data.SqlClient SQL Data provider where an attacker can perform an AiTM (adversary-in-the-middle) attack between the SQL client and the SQL server. This may allow the attacker to steal authentication credentials intended for the database server, even if the connection is established over an encrypted channel like TLS.
```

Comment:
Looks like there are additional explanation added to two of the security fixes comparing with  the original 2024 January blog post (https://devblogs.microsoft.com/dotnet/dotnet-framework-january-2024-security-and-quality-rollup/). Is that intentional?